pool/share.go

```diff
@@ -13,13 +13,32 @@ import (
 )
 
 // ShareWeights represents the associated weights for each known DCR miner.
-// With the share weight of the lowest hash DCR miner (LHM) being 1, the
-// rest were calculated as:
 //
-//	(Hash of Miner X * Weight of LHM)/ Hash of LHM
-var ShareWeights = map[string]*big.Rat{
-	CPU: new(big.Rat).SetFloat64(1.0), // Reserved for testing.
-}
+// The weights are calculated as:
+//
+//	Hash of Miner X / Hash of LHM
+var ShareWeights = func() map[string]*big.Rat {
+	// Find the lowest hash rate miner.  When there is not
```

Comment:
Incomplete comment

Reply:
Added a short circuit path for the empty case and commented it accordingly.